src/app/select-trajectory/select-trajectory.page.ts

```diff
@@ -150,6 +159,70 @@ export class SelectTrajectoryPage {
     this.router.navigate(['/diary'])
   }
 
+  async createExample() {
+    this.settingsService.getConfig('newApp').subscribe(
+      (firstTimeOpenApp) => (this.firstTimeOpenAppDefault = firstTimeOpenApp),
+      () => null,
+      () => {
+        this.userConfiguration = new UserConfiguration()
+        this.userConfiguration.hasFirstOpenApp =
+          this.firstTimeOpenAppDefault.defaultFirstTimeOpenApp
+      }
+    )
+    if (this.userConfiguration.hasFirstOpenApp) {
+      this.importExample()
+    }
+  }
+
+  setFirstTimeApp(consented: UserConfiguration) {
+    this.firstTimeOpenAppDefault.defaultFirstTimeOpenApp =
+      consented.hasFirstOpenApp
+    this.settingsService.saveConfig('newApp', this.firstTimeOpenAppDefault)
+  }
+
+  importExample() {
+    this.http.get('assets/trajectories/muenster.json').subscribe((res) => {
+      this.jsonDataResult = res
+    })
+
+    setTimeout(() => {
+      new Promise((resolve) => {
+        const json = JSON.stringify(this.jsonDataResult)
+        const trajectory =
+          this.trajectoryImportExportService.createTrajectoryFromImport(
+            json,
+            'Münster',
```

Comment:
I'm not quite sure, but shouldn't the title `Münster` come from the json-file itself instead of being hardcoded? If not, we should consider putting this into the translation-files, that way we would stay consistent (with all strings visible to the user being in one place).

Reply:
Add it to the translation files